.github/workflows/main.yml

```diff
@@ -23,10 +23,7 @@ jobs:
 
       - name: Build each example on every commit
         run: |
-          for example in example/pico-*; do
-            mkdir -p $example/build
-            cd $example/build
-            cmake ..
-            make
-            cd -
-          done
+          mkdir -p build
+          cd build
+          cmake ..
+          cmake --build . --target pico_blinky pico_blinky_cplusplus pico_empty pico_flash_io pico_fpga pico_hello_world pico_ssram
```

Comment:
This is the idea interface, permits to quickly compile an example to test something